src/main/java/org/icgc_argo/workflow/search/model/common/RunRequest.java

```diff
@@ -24,7 +24,10 @@
 import io.swagger.annotations.ApiModel;
 import java.util.HashMap;
 import java.util.Map;
-import lombok.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
```

Comment:
did you run optimize imports + mvn format and this wasn't optimized or was it optimized and intelij messed things up?

Reply:
I just ran optimize imports + mvn format. I guess this hasn't been run since the workflow-api refactor. Maybe it was on the TODO list after the refactor PR and I just forgot cause it was so long ago.

Reply:
I just ran it again for a sanity check, no changes